core/modules/wdb/QueryRunner.cc

```diff
@@ -314,24 +315,31 @@ void QueryRunner::_transmit(bool last, uint rowCount, size_t tSize) {
     if (!_cancelled) {
         // StreamBuffer::create invalidates resultString by using std::move()
         xrdsvc::StreamBuffer::Ptr streamBuf(xrdsvc::StreamBuffer::createWithMove(resultString));
-        bool sent = _task->sendChannel->sendStream(streamBuf, last);
-        if (!sent) {
-            LOGS(_log, LOG_LVL_ERROR, _task->getIdStr() << " Failed to transmit message!");
-        }
-        // Block on the buffer being sent.
-        auto totalBytes = xrdsvc::StreamBuffer::getTotalBytes();
-        LOGS(_log, LOG_LVL_INFO, _task->getIdStr() << " waiting for buffer largeResult=" << _largeResult
-                << " totalBytes=" << totalBytes);
+        _sendBuf(streamBuf, transmitHisto, "body");
+    } else {
+        LOGS(_log, LOG_LVL_DEBUG, "_transmit cancelled");
+    }
+    _largeResult = true; // Transmits after the first are considered large results.
+}
+
+
+void QueryRunner::_sendBuf(xrdsvc::StreamBuffer::Ptr& streamBuf,
+                           util::TimerHistogram& histo, std::string const& note) {
+    bool sent = _task->sendChannel->sendStream(streamBuf, false);
```

Comment:
Why `last` is always set to `false`? Was this a bug fix? In the previous version of the code a value of `last` was passed into the method from:
```
void QueryRunner::_transmit(bool last, ...)
```

Reply:
Thanks, fixed now.